contracts/clients/package.json

```diff
@@ -27,5 +27,9 @@
     "chai": "^4.3.4",
     "mocha": "^9.0.3",
     "source-map-support": "^0.5.20"
+  },
+  "resolutions": {
+    "ethers": "5.5.3",
```

Comment:
This is needed to make `deno` happy, it seems to have issues with multiple `ethers` versions being used. Will retest when other issues are resolved.